app/views/batch_contexts/_new_bc_form.erb

```diff
@@ -28,7 +28,7 @@
 
 
     <%= form.input :staging_location %>
-    <%= form.input :staging_style_symlink %>
+    <%= form.hidden_field :staging_style_symlink, value: false %>
```

Comment:
Can you put a comment here indicating why you are doing this?

Reply:
sure, i put a comment in above it